includes/admin/class-wc-payments-admin.php

```diff
@@ -420,6 +429,9 @@ public function add_payments_menu() {
 		$this->add_menu_notification_badge();
 		$this->add_update_business_details_task();
 		$this->add_disputes_notification_badge();
+		if ( $this->wcpay_gateway->get_option( 'manual_capture' ) === 'yes' ) {
```

Comment:
From stylistic PoV, the if statement can be moved inside 'add_transactions_notification_badge'.

Reply:
IMO, that's a potential violation of the single responsibility principle. `add_disputes_notification_badge` should do exactly that - add the badge.

Besides, having the conditions here makes it clear early on that adding the badge is dependent on conditions.

Also, a little above in the same method we have more such conditional UI rendering.

Let me know if you still think it's better to move the conditions inside the method.

Reply:
I think we can look into this later when we'll be dropping feature flags. Marking this as resolved.